source/features/user-choice-color.js

```diff
@@ -0,0 +1,11 @@
+export default function () {
+	const html = document.getElementsByTagName('html');
+	const newTweetButton = document.getElementById('global-new-tweet-button');
```

Comment:
Just use querySelector on these. You gain nothing by using the explicit methods.